external/harvestPackages/harvestPackages.netstandard1.6.depproj

```diff
@@ -1,146 +1,80 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <Import Project="$(ProjectDir)pkg/baseline/baseline.props" />
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
 
   <PropertyGroup>
     <TargetFrameworks>netstandard1.6</TargetFrameworks>
   </PropertyGroup>
 
+  <ItemDefinitionGroup>
+    <LibraryPackage>
+      <!--
+        Feed a higher version then any known stable version to ensure we get the latest stable
```

Comment:
I don't think this is needed see https://github.com/dotnet/buildtools/blob/master/src/Microsoft.DotNet.Build.Tasks.Packaging/src/GetLastStablePackage.cs#L129-L132